### PR TITLE
added AWS PrivateLink docs

### PR DESCRIPTION
**Pull request by esharf**
## Status
Ready.

## Related Issues
related: https://github.com/demisto/content/pull/25681

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]
Looks good

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 1]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
    * **Access Key**: add the Access key that you saved when creating the IAM user.
    * **Secret Key**: add the Secret key that you saved when creating the IAM user.
 1. Click **Test** to validate the connection (Keys & Token).
+
+### Working with PrivateLinks
+When using AWS PrivateLinks (sts/service links) you should make sure that the DNS hostname is configured properly.
+You can do it by one of the following methods
+1. [AWS Private DNS](https://docs.aws.amazon.com/vpc/latest/privatelink/manage-dns-names.html).
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:305]:
```suggestion
- [AWS Private DNS](https://docs.aws.amazon.com/vpc/latest/privatelink/manage-dns-names.html).
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 2]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+1. Using the python pass arguments configuration
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:306]:
```suggestion
- Using the python pass arguments configuration:
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 3]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+   * Go to **Settings** -> **About** -> **Troubleshooting** -> **Server Configuration**.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:307]:
```suggestion
   1. Go to **Settings** > **About** > **Troubleshooting** > **Server Configuration**.
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 4]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+   * Click on **Add Server Configuration**.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:308]:
```suggestion
   2. Click **Add Server Configuration**.
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 5]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+   * Add the key `python.pass.extra.keys`.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:309]:
```suggestion
   3. Add the key `python.pass.extra.keys`.
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 6]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+   * In the configuration value field `--add-host=<host>:X.X.X.X`.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:310]:
```suggestion
   4. In the configuration value field `--add-host=<host>:X.X.X.X`.
```

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 7]
```diff
@@ -298,3 +298,14 @@ The following steps will exemplify the creation of an IAM integration associated
...
+   * If you are using STS authentication than you might need to configure also the sts-host `--add-host=<host>:X.X.X.X##--add-host=<sts-host>:Y.Y.Y.Y`.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/aws-auth-guide.md:311]:
```suggestion
   5. If you are using STS authentication, you might need to also configure the sts-host `--add-host=<host>:X.X.X.X##--add-host=<sts-host>:Y.Y.Y.Y`.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Pull request merged**